ui/imports/shared/views/TransactionContextMenu.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.15
+
+import StatusQ.Popups 0.1
+
+StatusMenu {
+    id: root
+
+    required property string networkExplorerName
+
+    signal viewTxOnExplorerRequested
+    signal copyTxHashRequested
+
+    StatusAction {
+        objectName: "viewTxOnExplorerItem"
+        text: qsTr("View on ") + networkExplorerName
```

Comment:
```suggestion
        text: qsTr("View on %1").arg(networkExplorerName)
```

Reply:
Never cut sentences like this pls

Reply:
ah crap, working for too long in status-go does this to a man
fixed here https://github.com/status-im/status-desktop/pull/16762/commits/a0e683ea98bcc94d0aad56ca283172a2069fdf3e